packages/medusa-react/src/hooks/store/product-tags/queries.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  StoreGetProductTagsParams,
+  StoreProductTagsListRes,
+} from "@medusajs/medusa"
+import { Response } from "@medusajs/medusa-js"
+import { useQuery } from "@tanstack/react-query"
+import { useMedusa } from "../../../contexts"
+import { UseQueryOptionsWrapper } from "../../../types"
+import { queryKeysFactory } from "../../utils"
+
+const PRODUCT_TAGS_QUERY_KEY = `product_tags` as const
+
+export const productTagKeys = queryKeysFactory(PRODUCT_TAGS_QUERY_KEY)
```

Comment:
question(non-blocking): should creating/updating a product invalidate all productTagKeys since that's when they are created>

Reply:
Since this is part of the storefront API, I don't think it should be affected by products being created/updated since that is an admin-only action(s), and the two will never happen in the same project. It should however be the case for the admin version of this hook, which I don't think is the case atm 😄

Reply:
makes sense, in that case disregard this 😄